apps/atomy-q/API/app/Http/Controllers/Api/V1/RequisitionVendorSelectionController.php

```
@@ -80,7 +81,9 @@ public function update(Request $request, string $rfqId): JsonResponse
             ], 422);
         }
 
-        $rows = DB::transaction(function () use ($rfq, $tenantId, $vendorIds, $request, $vendors): Collection {
+        $decisionTrailRecorder = app(DecisionTrailRecorder::class);
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $decisionTrailRecorder. Keep variable name length under 20. (undefined)

(LongVariable)